oddbird/Metecho
# update splash page screenshot

## Cute animal pic
_Because everyone likes pictures of animals._

![images](https://user-images.githubusercontent.com/1581694/78080186-5607b380-7373-11ea-9c46-a583f8af7d03.jpg)


## Link to Trello card (if applicable)
_If this PR relates to a Trello card, don't forget to reference this PR on the card as well._

https://trello.com/c/BGQuFDrG/138-chore-update-splash-page-screenshots

## Description
_The commit messages say what you did; this should explain why and how._



## Steps to test/reproduce
_Please explain how to best reproduce the issue and/or test the changes locally._
log out to see login screen


## Impacted areas of website
_Provide URLs and/or a description of where to look._

homepage

## Show me
_Provide screenshots/animated gifs/videos if necessary._

![login-screen](https://user-images.githubusercontent.com/1581694/78080231-6d46a100-7373-11ea-9817-539584c41fed.jpg)


---

## Reviewer tasks:
